editor/test/integration/collection_test.clj

```diff
@@ -363,7 +363,7 @@
 
           (testing "Components in game object embedded inside collection."
             (let [collection (project/get-resource-node project collection-resource)
-                  game-object-instance (test-util/add-embedded-game-object! app-view project collection)
+                  game-object-instance (test-util/add-embedded-game-object! collection)
```

Comment:
The `test-util/add-embedded-game-object!` function no longer take an `app-view` and a `project` argument, and makes no attempt to update the selection in the `app-view`.